frontend/lib/ui/Form/useKeyboard.ts

```diff
@@ -0,0 +1,98 @@
+import * as React from "react";
+
+type Dir = "up" | "down" | "first" | "last";
+
+export function useKeyboard(innerRef: React.MutableRefObject<HTMLFormElement | null>) {
```

Comment:
I dont expect a hook named "useKeyboard" to only accept a specific "HTMLFormElement" ref.
(from the imports perspective you dont get the context of "form" eg. @kiesraad/ui)

This seems to be very specific form keyboard code.

Reply:
Absolutely, I've renamed it to `useFormKeyboardNavigation()` to better reflect what it does in https://github.com/kiesraad/abacus/pull/696/commits/cd14c54715f44f73ed93e8d43c76d4975b50cc5d